vendor/CMakeLists.adb.txt

```diff
@@ -147,6 +147,9 @@ add_library(libcrypto STATIC
 target_include_directories(libcrypto PUBLIC
 	core/libcrypto_utils/include boringssl/include)
 
+set(THREADS_PREFER_PTHREAD_FLAG ON)
+find_package(Threads REQUIRED)
```

Comment:
Wouldn't it be cleaner to do this once in `vendor/CMakeLists.txt`? Maybe after this line:

https://github.com/nmeum/android-tools/blob/903aa02f24c238b59605be344372c5c8a4e5c2f6/vendor/CMakeLists.txt#L39

🤔

Reply:
Fixed. Thanks!